arch/xtensa/src/esp32/esp32_spiflash.c

```diff
@@ -1933,6 +1934,15 @@ static int esp32_ioctl(FAR struct mtd_dev_s *dev, int cmd,
         }
         break;
 
+      case MTDIOC_ERASESTATE:
+        {
+          FAR uint8_t *result = (uint8_t *)arg;
```

Comment:
The cast need add FAR too:)

Reply:
That's the price for doing things in a hurry hehehe
Thanks, now it is properly fixed.